server/auth.go

```diff
@@ -41,12 +43,16 @@ type ClientAuthentication interface {
 
 // Accounts
 type Account struct {
-	Name    string
-	Nkey    string
-	mu      sync.RWMutex
-	sl      *Sublist
-	imports importMap
-	exports exportMap
+	Name     string
+	Nkey     string
+	mu       sync.RWMutex
+	sl       *Sublist
+	imports  importMap
+	exports  exportMap
+	nae      int
+	maxnae   int
+	maxaettl time.Duration
+	prunning bool
```

Comment:
I believe `prunning` is with 1 `n`

Reply:
I meant p-running. As in prune is running.

Reply:
I see ;-)

Reply:
Yours makes more sense though, will change.